server/sonar-server/src/main/java/org/sonar/server/rule/RubyRuleService.java

```diff
@@ -83,6 +83,11 @@ public Rule findByKey(String ruleKey) {
     query.setDebtCharacteristics(RubyUtils.toStrings(params.get("debtCharacteristics")));
     query.setHasDebtCharacteristic(RubyUtils.toBoolean(params.get("hasDebtCharacteristic")));
     query.setSortField(RuleNormalizer.RuleField.NAME);
+    String profile = Strings.emptyToNull((String) params.get("profile"));
+    if (profile != null) {
```

Comment:
instead of using emptyToNull you could have used in the test !Strings.isNullOrEmpty(profile). It's really a matter of personal taste, so not a big deal